src/applications/disability-benefits/686/config/form.js

```diff
@@ -269,41 +277,16 @@ function createLocationUISchemaForKey(
 ) {
   return {
     'ui:title': title,
-    countryDropdown: {
-      'ui:title': 'Country',
+    state: {
+      'ui:title': 'State (or country if outside the USA',
       'ui:required': isRequiredCallback,
```

Comment:
Can this property be removed if we're always asking for city and state (or country) moving forward?